modules/backlogs/app/assets/javascripts/backlogs/editable_inplace.js

```diff
@@ -31,7 +31,7 @@ RB.EditableInplace = (function ($) {
 
     displayEditor: function (editor) {
       this.$.addClass("editing");
-      editor.find(".editor").bind('keyup', this.handleKeyup);
+      editor.find(".editor").bind('keypress', this.handleKeypress).bind('keyup', this.handleKeyup);
```

Comment:
Instead of checking two different events here, just use `keydown`. This has several advantages: First, it ensures that the behavior you describe does not occur on "Escape". At the same time, it bypasses the [chrome bug](https://stackoverflow.com/a/5479683/8900797) that "Escape" does not fire on `keypress`. So we need significantly less code changes.

So just using `editor.find(".editor").bind('keydown', this.handleKeydown);` and renaming the method below should already be sufficient.

Reply:
I created new commit for this review.
Thank you for reviewing.